src/SUMMARY.md

```diff
@@ -1,34 +1,35 @@
 # Table of contents
 
-* [Introduction](README.md)
+- [Introduction](README.md)
 
 ## Domain registry essentials
 
-* [Architecture overview](domain-registry-essentials/architecture-overview.md)
-* [Namehashing](domain-registry-essentials/namehashing.md)
-* [Resolving domain records](domain-registry-essentials/resolving-domain-records.md)
-* [Records reference](domain-registry-essentials/records-reference.md)
-* [CNS smart contracts](domain-registry-essentials/cns-smart-contracts.md)
+- [Architecture overview](domain-registry-essentials/architecture-overview.md)
+- [Namehashing](domain-registry-essentials/namehashing.md)
+- [Resolving domain records](domain-registry-essentials/resolving-domain-records.md)
+- [Records reference](domain-registry-essentials/records-reference.md)
+- [CNS smart contracts](domain-registry-essentials/cns-smart-contracts.md)
```

Comment:
GitBook will change these back, once we return to it and start making edits there (this will happen when they fix their bugs with GitHub synchronization)